src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -267,8 +263,8 @@ public static void TestDecryptSimpleTripleDes_IssuerAndSerial()
         }
 
         [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
         [OuterLoop(/* Leaks key on disk if interrupted */)]
+        [ActiveIssue(3334, PlatformID.AnyUnix)]
```

Comment:
But... you have code for the implicit skid case.  I'd expect the test to only be disabled if you hadn't written the fallback...  And, likewise, that the test being disabled should preclude fallback code being added in this change.